pegasus/sites.v3/code.org/public/curriculum/game-design.haml

```diff
@@ -0,0 +1,71 @@
+---
+title: Game Design
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/game-design-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.overlay
+  .wrapper.grid-container.justify-space-between.align-items-center.gap-4.wider-text
+    .text-wrapper
+      %h1.white
+        =hoc_s("game_design_page.heading")
+      %p.body-two.white
+        =hoc_s("game_design_page.desc")
+      %a.link-button.white{href: "#curricula"}
+        =hoc_s("game_design_page.button.explore_curriculum")
+    %figure
+      %img{src: "/images/game-design/game-design-banner-graphic.png", alt: "", style: "width: 100%"}
+
+%section
+  .wrapper.centered
+    %h2=hoc_s("game_design_page.why.heading")
+    %p=hoc_s("game_design_page.why.desc")
+    = view :"game_design/game_design_why_list"
+    .button-wrapper.centered
+      %a.link-button{href: "#curricula"}
+        =hoc_s("game_design_page.button.explore_curriculum")
+      %a.link-button.secondary{href: "#"}
```

Comment:
Is this href intentionally "#"?

Reply:
Ah thank you for catching this! 🙏 Updated here: https://github.com/code-dot-org/code-dot-org/pull/59315/commits/b54aa7dca060daf82a1825bb12b8242dd58fd459